### PR TITLE
Added treatment_neuraminidase to ccpuk parser

### PR DESCRIPTION
**Pull request by pipliggins**

### CODE REVIEW

**Comment by abhidg** [comment]
thanks! could you re-write in TOML? I have merged the CCPUK toml into main

**Comment by pipliggins** [comment]
Sure.. It might also be useful to define some style rules for the toml file - I personally find some of the longer combinedType sections quite hard to follow at the moment - this for example
```
[visit.treatment_dexamethasone]
description = "Treatment with Dexamethasone"
combinedType = "firstNonNull"
[[visit.treatment_dexamethasone.fields]]
field = "corticost_cmtrt_type"

[visit.treatment_dexamethasone.fields.values]
1 = false
2 = false
3 = true
88 = false
[visit.treatment_dexamethasone.fields.if]
corticost_cmyn = 1
[[visit.treatment_dexamethasone.fields]]
field = "dexamethasone"
description = "Dexamethasone 6mg once per day?"

[visit.treatment_dexamethasone.fields.values]
1 = true
2 = false
3 = false
4 = true
5 = true
6 = true
```
might benefit from some indentation for readability (although I recognise that's mostly personal preference). We might also want to decide on some defined rules about where the square bracket style of defining things as used above is used vs this style.
```
[subject.date_death]
combinedType = "firstNonNull"
fields = [
    { field = "dsstdtc", description = "Outcome date", if = { dsterm = 4 } },
    { field = "dsstdtc_v2", description = "Outcome date", if = { dsterm_v2 = 4 } },
    { field = "flw_date_death", description = "Date of death" },
]
```

**Comment by abhidg** [comment]
Ok with indenting under second level. TOML doesn't allow line breaks within { } at the moment (will change in a future version of the spec), so having an max line length for { } style is good, 100/120? Yet to find an autoformatter which would be great to lint/autocorrect these kinds of things.

**Comment by pipliggins** [comment]
That sounds good, I'll keep an eye out for a formatter. I'm going to make this change in toml on another branch and reject this pull commit because that's easier than fighting all the conflicts now the json file has been deleted.
